ranger/data/scope.sh

```diff
@@ -160,7 +160,26 @@ handle_image() {
         #     exit 1;;
 
         ## PDF
-        # application/pdf)
+        application/pdf)
+            if [[ "$(uname)" = "Darwin" ]]; then
+                ## Use mac built-in quicklook to preview pdf
+                ## rename extension from .png to .jpg provided by IMAGE_CACHE_PATH
+                ## image viewer can determine the actual format
+                qlmanage -t -s "${DEFAULT_SIZE%x*}" \
+                         -o "${IMAGE_CACHE_PATH%/*}" \
+                         ${FILE_PATH} \
+                         && mv "${IMAGE_CACHE_PATH%/*}/${FILE_PATH##*/}.png" "${IMAGE_CACHE_PATH}" \
+                         && exit 6 || exit 1
+            fi
+            exit 1;;
+            #if [[ "$(uname)" = "Darwin" ]]; then
```

Comment:
I couldn't really get this to work. It works the first time a file is previewed but isn't respawned thereafter. I think we should use `exit 0`.

Reply:
Sorry. I haven't tried this for a while. Do you mean the commented code for spawning quicklook window? Indeed, it's not respawning. It seems it relies on cache and is not calling the code again. That's how ranger works. What should I do to make it call everytime?

Reply:
`exit 0`, that should make ranger show the cat output and try to regenerate it every time.